app/code/Magento/Cookie/Test/Unit/Controller/Index/NoCookiesTest.php

```diff
@@ -5,60 +5,68 @@
  */
```

Comment:
Could you please also declare the `declare(strict_types=1);`?
```suggestion
 */

declare(strict_types=1);
```